checkin/models.py

```diff
@@ -29,6 +56,14 @@ def __str__(self):
 
 
 class UserSkill(models.Model):
+    """Model for user's skill
+
+    :var profile: Profile model
+    :var skill: skill model
```

Comment:
This docstring does not provide any new information. At least describe the purpose of the model